<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials" tests="2" time="0.002" ignored="0" failures="0" timestamp="08 Jan 2018 14:54:11 GMT" skipped="1" hostname="ashishti" errors="1">
  <testcase name="setup" time="0.000" classname="com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.test.automation.uiAutomation.testBase.TestBase.getUrl(TestBase.java:57)
	at com.test.automation.uiAutomation.testBase.TestBase.init(TestBase.java:43)
	at com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials.setup(TC001_VerifyLoginWithInvalidCredentials.java:23)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.TestRunner.beforeRun(TestRunner.java:634)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setup -->
  <testcase name="verifyLoginWithInvalidCredentials" time="0.002" classname="com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials">
    <skipped/>
  </testcase> <!-- verifyLoginWithInvalidCredentials -->
</testsuite> <!-- com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials -->
